src/tile.scss

```diff
@@ -1,97 +1,657 @@
+    .#{$block}__content-byline {
+      @include fd-reset();
+
+      max-width: 100%;
+      font-size: var(--sapFontSmallSize);
+      line-height: 1.063rem;
+      color: var(--sapTile_TitleTextColor);
+    }
+
+    .#{$block}__content-text {
+      @include line-clamp();
+    }
+
+    .#{$block}__content-byline {
+      @include fd-ellipsis();
+
+      color: var(--sapTile_TextColor);
+    }
+
+    &.#{$block}--s {
+      .#{$block}__content {
+        .#{$block}__section {
+          &:nth-child(2) {
+            max-width: 8.25rem;
+          }
+        }
+      }
+    }
   }
 
-  &__actions {
-    @include fd-reset();
+  &--slide {
+    padding: 0;
+    display: flex;
+    align-items: flex-end;
+
+    .#{$block}__background-img {
+      @include fd-reset();
+
+      height: 100%;
+      width: 100%;
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background-size: cover;
+      border-radius: $fd-tile-border-radius;
+
+      @include fd-hover() {
+        opacity: 0.9;
+      }
+
+      @include fd-active() {
+        opacity: 0.8;
+      }
+    }
+
+    .#{$block}__toggle {
+      @include fd-reset();
+      @include set-height(3rem);
+      @include set-width(3rem);
+
+      display: none;
+      padding: 0.5rem;
+      position: absolute;
+      background: transparent;
+      top: $fd-tile-toggle-button-position;
+      right: $fd-tile-toggle-button-position;
+
+      @include fd-rtl() {
+        right: initial;
+        left: $fd-tile-toggle-button-position;
+      }
+
+      &::before {
+        @include set-height(2rem);
+        @include set-width(2rem);
+        @include fd-flex-center();
+
+        background-color: var(--sapBlockLayer_Background);
+        opacity: 0.6;
+        border-radius: 50%;
+        font-family: SAP-icons;
+        font-size: 1rem;
+        color: var(--sapButton_Emphasized_TextColor);
+        text-shadow: var(--sapButton_Emphasized_TextShadow);
+        content: "\e14b";
+      }
+
+      &--pause {
+        &::before {
+          content: "\e14c";
+        }
+      }
+
+      @include fd-focus() {
+        outline: none;
+      }
+
+      @include fd-hover() {
+        &::before {
+          opacity: 0.8;
+        }
+      }
+
+      @include fd-active() {
+        &::before {
+          opacity: 1;
+        }
+      }
+    }
+
+    .#{$block}__container {
+      @include fd-reset();
+      @include set-height(6.125rem);
+
+      @include fd-flex($direction: column) {
+        justify-content: space-between;
+      }
+
+      z-index: 2;
+      width: 100%;
+      max-width: 100%;
+      background-color: var(--sapBlockLayer_Background);
+      opacity: 0.8;
+      padding-top: 0.375rem;
+      padding-left: $fd-tile-padding;
+      padding-right: $fd-tile-padding;
+      border-bottom-left-radius: $fd-tile-border-radius;
+      border-bottom-right-radius: $fd-tile-border-radius;
+    }
+
+    .#{$block}__header {
+      @include set-height(3.5rem);
```

Comment:
Also this (3.625rem)
![image](https://user-images.githubusercontent.com/26483208/82341475-c2587800-99f0-11ea-9cef-ec12110f7352.png)
![image](https://user-images.githubusercontent.com/26483208/82341455-bbca0080-99f0-11ea-9ab9-c344145f7baf.png)